src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -14,20 +14,27 @@ public class Line extends BaseEntity {
     @Column(unique = true)
     private String name;
     private String color;
+    private int extraFare;
```

Comment:
```suggestion
    private Fare extraFare;
```

Fare 클래스를 만들었으면 여기에도 적용해야 하지 않을까요?